packages/dbml-parse/src/lib/parser/utils.ts

```diff
@@ -46,20 +46,25 @@ export function convertFuncAppToElem(
     return new None();
   }
 
-  const attributeList =
-    _.last(cpArgs) instanceof ListExpressionNode ? (cpArgs.pop() as ListExpressionNode) : undefined;
-
-  if (cpArgs.length === 3 && extractVariableNode(cpArgs[1]).unwrap().value === 'as') {
-    return new Some(
-      factory.create(ElementDeclarationNode, {
-        type,
-        name: cpArgs[0],
-        as: extractVariableNode(cpArgs[1]).unwrap(),
-        alias: cpArgs[2],
-        attributeList,
-        body,
-      }),
-    );
+  const attributeList = _.last(cpArgs) instanceof ListExpressionNode
+    ? (cpArgs.pop() as ListExpressionNode)
+    : undefined;
+
+  if (cpArgs.length === 3) {
+    const asKeywordNode = extractVariableNode(cpArgs[1]).unwrap_or(undefined);
```

Comment:
Can we use this instead?
```suggestion
    const asKeywordNode = extractVariableNode(cpArgs[1])?.value;
```

Reply:
Updated in https://github.com/holistics/dbml/pull/598/commits/d1b628cc6539e73403b9099bcfa6b03c94e5cbf6